ntbs-service/Pages/Shared/_AlertsAndActions.cshtml

```diff
@@ -26,7 +26,7 @@
                                     <br/>
                                     @alert.Action
                                 </div>
-                                @if (alert.AlertType != AlertType.DataQualityPotientialDuplicate)
+                                @if (alert.ActionLink != Context.Request.Path)
```

Comment:
This is potentially a little fragile as paths are a little ambiguous (e.g. adding/removing a forward slash at the end doesn't make a difference)

You could try and use the `Uri.Compare` method (see google for more info)

Reply:
Having thought about it a bit - I think this behaviour is fine. If the user does type in a nonsense path eg. notification/1/awdak then the 'take action' link appears, and clicking on it takes the user back to notification/1/, thus hiding the take action link again.